__tests__/unit/repositories/masking.repository.test.ts

```
@@ -210,4 +216,315 @@ describe('Masking Repository', () => {
+
+      mockHandlePostExecuteSqlStatement.mockResolvedValue({
+        rows: [{ id: mockInsertedId }],
+        rowCount: 1,
+      });
+
+      const result = await createMasking(mockMaskingData);
+
+      expect(result).toBe(mockInsertedId);
+      expect(mockHandlePostExecuteSqlStatement).toHaveBeenCalledWith(
+        expect.objectContaining({
+          text: expect.stringContaining('INSERT INTO trs_masking'),
```

Biome (2.4.14)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)